### DISCUSSION TITLE
Is live wallpaper dynamic or static?

### DISCUSSION THREAD

**Discussion by NOIGlory** [breezy-weather/breezy-weather#1512] [Q&A] [answered]
### Describe your idea

Does this wallpaper change after weather or remain static?
For example, does this wallpaper change when rain stops?
这个壁纸会不会随着天气的改变而改变？
比如说，雨停的时候壁纸会不会发生变化？

### Acknowledgements

- [x] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.3.0 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment, marked as answer]
You can configure it to be dynamic or static. Have a look at Settings > Widgets > Live wallpaper settings.
